src/client/shadows/dynamicshadowsrender.h

```diff
@@ -120,11 +121,12 @@ class ShadowRenderer
 	float m_shadow_map_max_distance;
 	float m_shadow_map_texture_size;
 	float m_time_day{0.0f};
-	float m_update_delta;
 	int m_shadow_samples;
 	bool m_shadow_map_texture_32bit;
 	bool m_shadows_enabled;
 	bool m_shadow_map_colored;
+	u8 m_map_shadow_update_frames; /* Use this number of frames to update map shaodw */
+	u16 m_current_frame{0}; /* Current frame */
```

Comment:
no point to use a bigger type since `m_current_frame <= m_map_shadow_update_frames`
```suggestion
	u8 m_current_frame{0}; /* Current frame */
```

Reply:
I'd suggest to put a condition when incrementing m_current_frame, to avoid surprises with overflow when conditions for updating light frustum change.